js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js

```diff
@@ -80,8 +84,13 @@ define([
             this.data = ko.observable([]);
             this.domains = ko.observableArray();
             this.filterList = ko.observableArray([]);
+            this.selectedItems = ko.computed(() => {
+                filterUtils.getSelectedFilterValues(this.filterList());
```

Comment:
This arrow function always returns nothing. Is this a mistake or such behavior by design?

Reply:
fixed